Lib/argparse.py

```diff
@@ -1268,8 +1268,14 @@ def __call__(self, string):
 
         # all other arguments are used as file names
         try:
-            return open(string, self._mode, self._bufsize, self._encoding,
+            file = open(string, self._mode, self._bufsize, self._encoding,
```

Comment:
`file` is a builtin function, please use other name here. Suggested: `fh` or `file_handle`.